_episodes/01-introduction.md

```diff
@@ -11,5 +11,121 @@ keypoints:
 ---
 The example-based nature of [Matplotlib documentation](https://matplotlib.org/) is GREAT.
 
-{% include links.md %}
+Matplotlib is the standard when it comes to making plots in Python. It is versatile and allows for lots of functionality and different ways to produce many plots.
+We will be focusing on using matplotlib for High Energy Physics.
+
+# A simple example
+A with any python code it is always a good practice to import the necessary libraries as a first step.
+
+```python
+import matplotlib.pyplot as plt
+```
+
+Matplotlib graphs your data on Figures (i.e., windows, Jupyter widgets, etc.), each of which can contain one or more Axes (i.e., an area where points can be specified in terms of x-y coordinates, or theta-r in a polar plot, or x-y-z in a 3D plot, etc.). The simplest way of creating a figure with an axes is using pyplot.subplots. We can then use Axes.plot to draw some data on the axes:
```

Comment:
Is it better to say
"a figure with an axes" --> "a figure with axes" ?
Unless you mean an `Axes` object, which I would then write as the object.

Reply:
Actually this is copy pasted from https://matplotlib.org/stable/tutorials/introductory/usage.html#a-simple-example. 
So I would blame the Matplotlib documentation 😆 but I get what you mean.

I would assume they meant to write 
> The simplest way of creating a figure with an Axes *object* is using `pyplot.subplots`. We can then use `Axes.plot` to draw some data on the axes: